bundles/3scale-operator/bundles.yaml

```diff
@@ -21,3 +21,5 @@ bundles:
       image: quay.io/integreatly/3scale-bundle:v0.8.4
     - name: 3scale-operator.v0.9.0
       image: quay.io/integreatly/3scale-bundle:v0.9.0
+    - name: 3scale-operator.v0.10.0-mas
+      image: quay.io/integreatly/delorean:3scale-bundle-0.10.0mas-1
```

Comment:
@MStokluska @adam-cattermole will the `-1` be removed in the case of a GA'd image?

Reply:
For a GA'd image the tag used is the version from the CSV, so it will be quay.io/integreatly/3scale-bundle:v0.10.0-mas

Reply:
Actually, that is making me think, didn't we want these PRs to generate once it's GA for mas?

Reply:
The issue was that a testing cluster is needed by QE pre-GA. So we mirror map and generate private bundles / index for pre-GA images. On GA we just re-tag the image to the public repo and remove the mirror mapping.

Reply:
`For a GA'd image the tag used is the version from the CSV, so it will be quay.io/integreatly/3scale-bundle:v0.10.0-mas` thanks Adam, didn't realized it's going to work like that, good to know!